templates/features/high-availability.html

```diff
@@ -0,0 +1,87 @@
+{% extends 'features/features_layout.html' %}
+
+{% block title %}High availability (HA){% endblock title %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1G6FijpahZCSfxcuFAkv22wKCILEkF2u4Q0TQUfkW5i4/edit#{% endblock meta_copydoc %}
+
+{% block meta_description %}MicroK8s is the simplest production-grade upstream K8s. Lightweight and focused. Single command install on Linux, Windows and macOS. Made for DevOps, great for edge, appliances and IoT. Full high availability Kubernetes with autonomous clusters.{% endblock %}
+
+
+{% block features_content %}
+  <section class="p-strip u-no-padding--top">
+    <div class="row">
+      <h1>High availability (HA)</h1>
+    </div>
+    <div class="row">
+      <div class="col-6">
+        <p class="p-heading--three">Automatic, autonomous and self-healing HA</p>
+        <p>
+         Mission-critical applications require a resilient Kubernetes cluster, which can lose any node and still provide reliable services. Zero downtime is a necessity in production environments. Fault tolerance is especially critical for remote, unattended clusters, appliances and industrial IoT, where access is limited and distributed across a country, a continent or the globe.
+        </p>
+        <p>
+         MicroK8s makes high availability resilient and self-healing with no administrative interventions required.
+        </p>
+      </div>
+      <div class="col-4 u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/9d785fad-Automatic+node+allocation.svg" width="230" alt="Automatic node allocation"/>
+      </div>
+    </div>
+    <div class="row">
+      <h2>What is high availability Kubernetes?</h2>
+      <div class="col-8">
```

Comment:
I don't think you need this. The content is already constrained by the base template